### PR TITLE
chore: add `lazy-loading.cy.ts`

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/demo-cypress/src/tests/lazy-loading.cy.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+            expect(img.style.animationName).to.equal('tuiSkeletonVibe');
+
+            component.src =
+                'data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/4QDeRXhpZgAASUkqAAgAAAAGABIBAwABAAAAAQAAABoBBQABAAAAVgAAABsBBQABAAAAXgAAACgBAwABAAAAAgAAABMCAwABAAAAAQAAAGmHBAABAAAAZgAAAAAAAABIAAAAAQAAAEgAAAABAAAABwAAkAcABAAAADAyMTABkQcABAAAAAECAwCGkgcAFgAAAMAAAAAAoAcABAAAADAxMDABoAMAAQAAAP//AAACoAQAAQAAAAEAAAADoAQAAQAAAAEAAAAAAAAAQVNDSUkAAABQaWNzdW0gSUQ6IDY3Nf/bAEMACAYGBwYFCAcHBwkJCAoMFA0MCwsMGRITDxQdGh8eHRocHCAkLicgIiwjHBwoNyksMDE0NDQfJzk9ODI8LjM0Mv/bAEMBCQkJDAsMGA0NGDIhHCEyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMv/CABEIAAEAAQMBIgACEQEDEQH/xAAVAAEBAAAAAAAAAAAAAAAAAAAABf/EABUBAQEAAAAAAAAAAAAAAAAAAAAB/9oADAMBAAIQAxAAAAGOE//EABQQAQAAAAAAAAAAAAAAAAAAAAD/2gAIAQEAAQUCf//EABQRAQAAAAAAAAAAAAAAAAAAAAD/2gAIAQMBAT8Bf//EABQRAQAAAAAAAAAAAAAAAAAAAAD/2gAIAQIBAT8Bf//EABQQAQAAAAAAAAAAAAAAAAAAAAD/2gAIAQEABj8Cf//EABQQAQAAAAAAAAAAAAAAAAAAAAD/2gAIAQEAAT8hf//aAAwDAQACAAMAAAAQC//EABQRAQAAAAAAAAAAAAAAAAAAAAD/2gAIAQMBAT8Qf//EABQRAQAAAAAAAAAAAAAAAAAAAAD/2gAIAQIBAT8Qf//EABQQAQAAAAAAAAAAAAAAAAAAAAD/2gAIAQEAAT8Qf//Z';
+            fixture.detectChanges();
```
Comment (nsbarsukov) [inline projects/demo-cypress/src/tests/lazy-loading.cy.ts:68]: We can just add simple button inside template

```html
<button (click)="updateSrc()">click</button>
```

And just call
```ts
cy.get('button').click();
```

I hope it is more robust test and not such artificial.
What do you think about it ? 🤔

File: projects/demo-cypress/src/tests/lazy-loading.cy.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+            cy.wait(100).then(() => {
+                expect(img.style.animationName).to.equal('');
```
Comment (nsbarsukov) [inline projects/demo-cypress/src/tests/lazy-loading.cy.ts:71]: Could we use the following auto-retrying assertion instead ?

```ts
cy.get('img').should('have.css', 'animation-name', '')
```

File: projects/demo-cypress/src/tests/lazy-loading.cy.ts (hunk 1 above)
Comment (splincode) [reply projects/demo-cypress/src/tests/lazy-loading.cy.ts]: fixed

File: projects/demo-cypress/src/tests/lazy-loading.cy.ts (hunk 2 above)
Comment (splincode) [reply projects/demo-cypress/src/tests/lazy-loading.cy.ts]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**